rolling/distribution.yaml

```diff
@@ -2514,6 +2514,19 @@ repositories:
       url: https://github.com/ros-perception/pcl_msgs.git
       version: ros2
     status: maintained
+  perception_open3d:
+    release:
+      packages:
+      - open3d_conversions
+      tags:
+        release: release/rolling/{package}/{version}
+      url: https://github.com/ros-gbp/perception_open3d-release.git
```

Comment:
FYI, we highly recommend that you use a release repository in https://github.com/ros2-gbp for Rolling.  If you don't do this, then this package can't automatically be rolled over into the next distribution.

I won't outright block on this, but let me know explicitly if you want to stay with your current release repository.

Reply:
https://github.com/ros-perception/perception_open3d/issues/24 conversation is in the works, I need access to the org to do so. I'm a member of the org but not admin to move / create new repositories which I need to transfer this project (and probably several others in the future)

Reply:
While GitHub redirection will work, I'm not sure whether our tools will follow that properly.  @nuclearsandwich can answer that.  The easiest thing to do here is to move it over and just use the `ros2-gbp` repository, that way there is no question.  I'll see what I can do about getting https://github.com/ros-perception/perception_open3d/issues/24 resolved.

Reply:
OK